src/components/standalone/hotspot/SettingsContent.vue

```diff
@@ -161,7 +164,10 @@ async function getListDevices() {
         label: item
       }))
       configurationForm.value.networkDevice = res.data.devices[0]
-    } else emptyDevices.value = true
+    } else {
+      isError.value = true
+      emptyDevices.value = true
```

Comment:
With `emptyDevices.value = true` the error notification will appear, but there is no title and description set (`error.value.empty_network_device` and `error.value.empty_network_device_description`)

Reply:
My bad, title and description are passed directly to the notification component